main.go

```diff
@@ -117,4 +117,9 @@ func main() {
 	paymentGetResp, err := client.GetPayment(paymentCreateResp.PaymentID)
 	handleError(err)
 	fmt.Println("Payment get response:", paymentGetResp)
+
+	// POST /webhook_verification_key/get
+	webhookVerificationKeyGetResp, err := client.GetWebhookVerificationKey("6c5516e1-92dc-479e-a8ff-5a51992e0001")
```

Comment:
⚠️ should this be a hardcoded value?

Reply:
Yep! These are well-known constant public key IDs. This one is the sandbox one. The alternative is figuring out how to get a webhook listener in the tests and fire a real webhook that is received by the CI machine in order to extract this value.